bpf/lib/nodeport.h

```diff
@@ -1507,6 +1548,30 @@ static __always_inline int dsr_reply_icmp4(struct __ctx_buff *ctx,
 		.tot_len	= bpf_htons((__u16)tot_len),
 	};
 
+	struct iphdr inner_ip_hdr __align_stack_8 = *ip4;
+	__be16 dport;
+	__s32 l4_dport_offset;
+
+	/*
+	 * DSR changes the destination address from service ip to pod ip and
```

Comment:
same nit about the multiline comment format consistency (see previous for details)